Phoenix/PHContext.m

```diff
@@ -136,10 +136,17 @@ - (void) setupAPI {
     self.context[@"Mouse"] = [PHMouse class];
     self.context[@"App"] = [PHApp class];
     self.context[@"Window"] = [PHWindow class];
-
+    
     self.context[@"require"] = ^(NSString *path) {
 
-        path = [weakSelf resolvePath:path];
+        if ([path isAbsolutePath]) {
```

Comment:
Primarily because the `else if ` branch calls resolvePath. I can bring it in and avoid the recursive call.